set.mm

```diff
@@ -91095,7 +91095,7 @@ is that it denies the existence of a set containing itself ( ~ elirrv ).
     ( cvv wcel wa wn cep wfr zfregfr efrn2lp mpan elex anim12i con3i pm2.61i )
     ACDZBCDZEZABDZBADZEZFZCGHRUBCICABJKUARSPTQABLBALMNO $.
 
-  $( A class is not an element of another class and vice versa.  (Contributed
+  $( Two classes are not elements of each other simultaneously.  (Contributed
      by AV, 5-Nov-2023.) $)
   elnanel $p |- ( A e. B -/\ B e. A ) $=
```

Comment:
I think that the alternative denial operator should be very seldom used outside of the NAND section. Here, elnanel is simply a rewriting of en2lp using a less standard and lesser known connective. There are many (many) theorems where NAND, NOR and XOR could be used to save a few symbols, but this would be at the cost of increasing the number of statements and the knowledge required from readers.

Maybe it is justified here since it serves as a basic example for Godel codes. If that's the case and we want to keep it, then we should probably add a comment to that effect and discourage its usage.

Reply:
Done